test/utils/image/manifest.go

```diff
@@ -0,0 +1,141 @@
+/*
```

Comment:
As we are maintaining only docker images in `test/images` so IMO it will be more relevant in `test/utils/images` directory.